app/queries/decidim/admin/user_admin_filter.rb

```diff
@@ -48,28 +48,58 @@ def filter_by_search_text(users)
         return users if search_text.blank?
 
         containing_proces_name = "%#{search_text}%"
-        users.where(<<-EOSQL, current_locale, containing_proces_name, current_locale, containing_proces_name)
-          (id in (select decidim_user_id
-                  from decidim_participatory_process_user_roles
-                  where decidim_participatory_process_id in (
-                    select id
-                    from decidim_participatory_processes
-                    where lower(title->>?) like lower(?)))
-          or id in  ( select decidim_user_id
-                      from decidim_assembly_user_roles
-                      where decidim_assembly_id in (
-                        select id
-                        from decidim_assemblies
-                        where lower(title->>?) like lower(?))))
-        EOSQL
+
+        if defined?(Decidim::Conferences)
```

Comment:
better avoid having this if-else structure, what will we do when we have to consider another space, like initiatives?
Can we construct the sql string before invoking the `where` and conditionaly concatenate the conferences part?